maps-app/src/main/java/com/esri/android/mapsapp/MapFragment.java

```diff
@@ -1611,7 +1606,13 @@ public void onLongPress(MotionEvent e) {
 		@Override
 		public boolean onRotate (MotionEvent e, double angle){
 			super.onRotate(e,angle);
-
+			// Show the compass if we're not pointed north
+			if (mMapView.getMapRotation()!= 0){
+				mCompass.setVisibility(View.VISIBLE);
+				mCompass.setRotationAngle(mMapView.getMapRotation());
+			}else{
+				mCompass.setVisibility(View.GONE);
+			}
```

Comment:
@zinfin Should these be considered in lieu of the compass PR?  The updates are similar, if not exact per formatting.